MailTests/MailboxManagerTests.swift

```diff
@@ -23,46 +23,48 @@ import InfomaniakLogin
 @testable import MailCore
 import XCTest
 
+// TODO: Fixme, DI setup missing
```

Comment:
In order to fix this test, maybe I should bring back the work on done on drive to build a DI environment suitable for tests.